src/Main/Main.java

```diff
@@ -0,0 +1,186 @@
+package Main;
+
+import Address.*;
+import Book.*;
+import Users.*;
+import Users.Client;
+
+import java.sql.SQLOutput;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Scanner;
+
+
+public class Main {
+
+
+    public static void main(String[] args) throws ISBN.ISBNFormatException {
+        //a valid ISBN for testing: 0569934052
+        MainService service = MainService.getInstance();
+        Scanner read = new Scanner(System.in);
+        int option;
+        do {
+            int option1;
+            System.out.println("1 - Librarian");
+            System.out.println("2 - Client");
+            option = read.nextInt();
+            switch (option) {
+                case (1): {
+                    do {
+                        System.out.println("1 - Creare cont nou");
+                        System.out.println("2 - Logare in cont existent");
+                        System.out.println("3 - Add a new book");
+                        System.out.println("4 - Edit a book");
```

Comment:
as a suggestion for improvements, you shouldn't display all the options until the user is logged in, but for what we want now is ok like this as well